setup.cfg

```diff
@@ -1,10 +1,9 @@
 [versioneer]
 VCS = git
 style = pep440
-versionfile_source = version/_version.py
-versionfile_build = version/_version.py
+versionfile_source = src/_version.py
+versionfile_build = src/_version.py
 tag_prefix =
-parentdir_prefix = python-ci-template
```

Comment:
So it is fine to skip the parentdir_prefix? I thought was needed for something else. @nromeoarena can you double check this?

Reply:
> So it is fine to skip the parentdir_prefix? I thought was needed for something else. @nromeoarena can you double check this?

Ufficial documentation says:

parentdir_prefix:
a optional string, frequently the same as tag_prefix, which appears at the start of all unpacked tarball filenames. If your tarball unpacks into 'myproject-1.2.0', this should be 'myproject-'. To disable this feature, just omit the field from your setup.cfg.

Reply:
I agree with @nromeoarena.
I checked the same documentation and came to the conclusion that this line is not required.